### ISSUE TITLE
Rebrand `unittest` project to `xunit`

### ISSUE THREAD

**Issue by peterblazejewicz** [OmniSharp/generator-aspnet#876]
The RC contains 2 different flavours of test framework: MSTest and xUnit:
```bash
Templates                            Short Name       Language      Tags          
----------------------------------------------------------------------------------
Console Application                  console          [C#], F#      Common/Console
Class library                        classlib         [C#], F#      Common/Library
Unit Test Project                    mstest           [C#], F#      Test/MSTest   
xUnit Test Project                   xunit            [C#], F#      Test/xUnit    
Empty ASP.NET Web Application        web              [C#]          Web/Empty     
MVC ASP.NET Web Application          mvc              [C#], F#      Web/MVC       
Web API ASP.NET Web Application      webapi           [C#]          Web/WebAPI    
Nuget Config                         nugetconfig                    Config  
```
The generator supports only `unittest`, so first migrate `unittext` to `xunit`, merge into `vs2017-rc` branch and only after this migrate (now) `xunit` project to `.cproj` and add `mstest` project.

Thanks!

**Comment by peterblazejewicz** [comment]
closed via: 4349436bc7db3b1316d0abc9dea1d589a869c21c
